recipes/labmaze/unvendor_deps.patch

```diff
@@ -0,0 +1,187 @@
+From 63bb67fd9ed5991bc1478241925d3d7e463d674d Mon Sep 17 00:00:00 2001
+From: Silvio Traversaro <silvio.traversaro@iit.it>
+Date: Tue, 13 Dec 2022 17:10:32 +0100
+Subject: [PATCH] unvendor_patch
+
+---
+ WORKSPACE                   | 17 +++++++----------
+ bazel/system_absl.BUILD     | 22 ++++++++++++++++++++++
+ bazel/system_pybind11.BUILD | 23 +++++++++++++++++++++++
+ labmaze/cc/BUILD            |  4 ++--
+ setup.py                    | 28 ++++++++++++++++++++++++++--
+ 5 files changed, 80 insertions(+), 14 deletions(-)
+ create mode 100644 bazel/system_absl.BUILD
+ create mode 100644 bazel/system_pybind11.BUILD
+
+diff --git a/WORKSPACE b/WORKSPACE
+index b997d5d..7afdbc0 100644
+--- a/WORKSPACE
++++ b/WORKSPACE
+@@ -24,11 +24,10 @@ http_archive(
+     url = "https://github.com/bazelbuild/bazel-skylib/archive/1.3.0.zip",
+ )
+ 
+-http_archive(
++new_local_repository(
+     name = "com_google_absl",
+-    sha256 = "54707f411cb62a26a776dad5fd60829098c181700edcd022ea5c2ca49e9b7ef1",
+-    strip_prefix = "abseil-cpp-20220623.1",
+-    url = "https://github.com/abseil/abseil-cpp/archive/20220623.1.zip",
++    path = "/home/traversaro/mambaforge/envs/labmazedev/include",  # absl placeholder
++    build_file = "@//bazel:system_absl.BUILD"
```

Comment:
How can this local path work in CI, or is this simply never built...?

Reply:
This is replaced by setup.py, following the structure already used in the recipe for the python headers, however given that a random local path was confusing, I substituted it with a clearly fake path.